src/bentoml/_internal/cloud/deployment.py

```diff
@@ -774,6 +774,8 @@ def _init_deployment_files(
         if setup_md5 != pod_files.get("setup.sh", ""):
             upload_files.append(("setup.sh", setup_script))
         self.upload_files(upload_files, console=console)
+        # Upload a ready flag file after all files are uploaded
+        self.upload_files([(".project_ready", b"")], console=console)
```

Comment:
```suggestion
        self.upload_files([(".bento_project_ready", b"")], console=console)
```

Reply:
it's also changed in yatai-serverless side so i will stick to it